mindjunkies/home/tests/test_views.py

```
@@ -0,0 +1,88 @@
+    return User.objects.create_user(username="testuser", password="password123")
+
+
+@pytest.fixture
+def auth_client(client, create_user):
+    client.force_login(create_user)
+    return client
+
+
+def test_home_view_unauthenticated(client):
+    """Test home view for an unauthenticated user (should see all courses)."""
+    new_courses = [
```

Ruff (0.8.2)

Local variable `new_courses` is assigned to but never used

Remove assignment to unused variable `new_courses`

(F841)